graph/api/api_test.go

```diff
@@ -203,12 +151,15 @@ func mockNamespaceGraph(t *testing.T) (*prometheus.Client, *prometheustest.PromA
 		"request_protocol":               "http",
 		"response_code":                  "200",
 		"grpc_response_status":           "0",
-		"response_flags":                 "-"}
+		"response_flags":                 "-",
+	}
 	q1m1 := model.Metric{
 		"source_workload_namespace":      "unknown",
 		"source_workload":                "unknown",
 		"source_canonical_service":       "unknown",
 		"source_canonical_revision":      "unknown",
+		"source_cluster":                 "east",
```

Comment:
This should be `unknown`.   It's unlikely a cluster would be set on an unknown source.